src/Tribe/Utils/Post_Thumbnail.php

```diff
@@ -155,44 +157,58 @@ public function fetch_data() {
 			return $this->data;
 		}
 
-		$post_id      = $this->post_id;
 		$image_sizes  = $this->get_image_sizes();
 		$thumbnail_id = $this->thumbnail_id;
 
-		$thumbnail_data = array_combine(
-			$image_sizes,
-			array_map(
-				static function ( $size ) use ( $thumbnail_id ) {
-					$size_data = wp_get_attachment_image_src( $thumbnail_id, $size );
+		$cache_key = empty( $thumbnail_id ) ? -1 : $thumbnail_id;
+
+		if ( empty( $cache_thumbnail[ $cache_key ] ) ) {
+			$thumbnail_data = array_combine(
+				$image_sizes,
+				array_map(
+					static function( $size ) use ( $thumbnail_id ) {
+						static $cache_size_data = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 8641151267c3c5461e2d34586d35cbfc</sub>